prisma/schema.prisma

```diff
@@ -26,4 +26,48 @@ model User {
   updatedAt DateTime @updatedAt
 
   @@index([email])
+}
+
+model Tournament {
+  id           Int      @id @default(autoincrement())
+  createdAt    DateTime @default(now())
+  updatedAt    DateTime @updatedAt 
+  rounds       Round[]
+  players      Player[]
+  status       String  @default("registration")
```

Comment:
can you change this to be an [ENUM](https://www.prisma.io/docs/orm/prisma-schema/data-model/models#defining-enums)? (this and all other statuses)

Reply:
Can we add a tournament name?